docs/demo.md

```diff
@@ -1,163 +1,19 @@
-
-```
-$ coda client send-txn -amount <AMOUNT> -privkey-path ./funded-wallet/key -receiver <RECEIVER_ADDRESS>
-```
-
-Replacing the `<AMOUNT>` argument with an integer value and the `<RECEIVER_ADDRESS>` with the public key of the account you're sending money to. It will prompt you for the private key password but it is blank.
-
-#### Check account balances
-
-Naturally, you might be interested in seeing how and when the network processes your payments. You can do that by either inquiring for a specific address or all account balances. Here are the commands to do so:
-
-**Checking account balance**
-```
-$ coda client get-balance -address <PUBLIC_KEY>
-```
-
-**Checking everyone's account balances**
-```
-$ coda client get-public-keys -with-balances
-```
-
-#### Add additional nodes to the network
-
-To spin up additional nodes, you'll need to point them to existing, running nodes. You can see how we spin up the nodes used in your network by reading through `./scripts/cluster.sh`, and if you want to spin up new nodes, you can follow the patterns there.
-
-Then, you can check whether this node is up and running with the following status command:
+The public key of the demo account is `4vsRCVMNTrCx4NpN6kKTkFKLcFN4vXUP5RB9PqSZe1qsyDs4AW5XeNgAf16WUPRBCakaPiXcxjp6JUpGNQ6fdU977x5LntvxrSg11xrmK6ZDaGSMEGj12dkeEpyKcEpkzcKwYWZ2Yf2vpwQP`, with the following private key file (the password is the empty string):
 
 ```
-$ coda client status -daemon-port <PORT>
+{"box_primitive":"xsalsa20poly1305","pw_primitive":"argon2i","nonce":"7S1YA5PinXhnLgLJ3xemVnVPWdJdhKZ9RSNQbns","pwsalt":"AzDoECCYyJL8KuoB2vrsVc9Wg3xJ","pwdiff":[134217728,6],"ciphertext":"5UQuiQVbXPmR63ikri792dWR6Dz5dYZm8dLzwDyqWovdP5CzrLY6Fjw3QTHXA9J3PDkPZpvhrQfGkgU81kr9184dfoJDhn5EXxJMCAM44SZdmBYVszEQaSQnyy4BwsbRXmfjBMSW9ooGu2a5dFi5KHX5na6fr62VUB"}
 ```
 
+This account has 100% of the stake.
 
-## Appendix
-
-This will show you how to get up and running with an appropriate Linux distribution on Google Cloud. We'll cover a few steps: 1) getting the instance set up, 2) interacting with the machine on the command line.
-
-#### Setting up the instance
-
-To get the instance set up, follow [these instructions](https://cloud.google.com/compute/docs/quickstart-linux), with the following changes:
-
-- **Choose `Ubuntu 18.04 LTS` as the OS.**
-- **Instead of 1 vCPU, choose to have 4 vCPUs**
-- **If you're on the west coast, instead of choosing us-east1 choose a west coast region**
-- **Change the amount of ram to 12GB**
-
-#### Interacting with the CLI
-
-To interact with the CLI, go to the compute engine -> VM instances page.
-
-Then, select the VM, start the VM, and click the SSH button to open up a terminal window.
-
+The demo container will run a block producer and a snark worker. You need to
+make sure the `--publish` at least the GraphQL port, for example `docker run
+--publish 3085:3085 -it codaprotocol/coda-demo`. Any additional arguments will
```

Comment:
FWIW when I run this, I get
```
$ docker run --publish 3085:3085 -it codaprotocol/coda-demo
Unable to find image 'codaprotocol/coda-demo:latest' locally
docker: Error response from daemon: manifest for codaprotocol/coda-demo:latest not found: manifest unknown: manifest unknown.
```

works when I put something like codaprotocol/coda-demo:0.0.12-beta-fix-coda-demo-0308cf7 though